### DISCUSSION TITLE
Advice for students contemplating a career challenging the current cosmogonical paradigm

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#121] [General]
*"Publishing Theoretical Physics in The Age of Censorship"*
[https://thescientificworldview.blogspot.com/2023/08/publishing-theoretical-physics-in-age.html](https://thescientificworldview.blogspot.com/2023/08/publishing-theoretical-physics-in-age.html)
by Glenn Borchardt

Interesting read but not good advice IMO; only a very narrow view of how the academic world works.

There is no such thing as a *career in challenging the current paradigm*.  If that student wants to get published, the first step is to learn mainstream physics, show he understands it (PhD, papers, reputation, etc...) and then he can take apart mainstream physics bit by bit in papers that will be accepted for publications (e.g. [Martín López-Corredoira](https://lopez-corredoira.com/ciencia_en.html), [Stacy McGaugh](http://astroweb.case.edu/ssm/pubs.html), [Pavel Kroupa](https://astro.uni-bonn.de/~pavel/), [Halton Arp](http://www.haltonarp.com/articles)...)

**Comment by HanDeBruijn** [comment]
![integrity2](https://github.com/a-cosmology-group/acg/assets/135433222/484b1efb-8087-4f91-8a24-55718dc147ae)
